Library/Formula/gradle.rb

```diff
@@ -1,11 +1,16 @@
 class Gradle < Formula
   desc "Build system based on the Groovy language"
   homepage "https://www.gradle.org/"
-  url "https://downloads.gradle.org/distributions/gradle-2.6-bin.zip"
-  sha256 "18a98c560af231dfa0d3f8e0802c20103ae986f12428bb0a6f5396e8f14e9c83"
+  url "https://downloads.gradle.org/distributions/gradle-2.7-bin.zip"
+  sha256 "cde43b90945b5304c43ee36e58aab4cc6fb3a3d5f9bd9449bb1709a68371cb06"
 
   def install
     libexec.install %w[bin lib]
     bin.install_symlink libexec+"bin/gradle"
   end
+
+  test do
+    output = shell_output("#{bin}/gradle --version")
+    assert output.include?("Gradle #{version}")
```

Comment:
We use `assert_match` over `assert output.include?` these days. Should be something like:

```
output = shell_output("#{bin}/gradle --version")
assert_match /Gradle #{version}/, output
```